library/build.gradle

```diff
@@ -20,8 +20,10 @@ android {
 }
 
 dependencies {
-    compile fileTree(dir: 'libs', include: ['diffutils-1.2.1.jar'])
-    compile 'com.android.support:design:23.1.1'
-    compile 'com.android.support:recyclerview-v7:23.1.1'
-    compile 'com.github.TonicArtos:SuperSLiM:ed0ba4b4d2'
+    compile fileTree(dir: 'libs', include: ['*.jar'])
+    compile 'com.android.support:design:23.3.0'
+    compile 'com.android.support:recyclerview-v7:23.3.0'
+    compile 'com.googlecode.java-diff-utils:diffutils:1.3.0'
+    compile 'com.tonicartos:superslim:0.4.13'
```

Comment:
Nice! Yea, I used the specific commit at the time because Master included a fix I needed.